recipes/pyside2ads/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "3.8.2.1" %}
+package:
+  name: pyside2ads
+  version: {{ version }}
+
+source:
+  url: https://github.com/metgem/PySide2Ads/archive/v{{ version }}.tar.gz
+  sha256: 39b75119212dbc465590f11b5e36909949fcc34006d96b843205a4971e8c4bde
+  patches:
+    - 0001-libcxx.patch              # [linux]
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  script_env:
+    - CMAKE_GENERATOR_PLATFORM=      # [win]
+    - CMAKE_GENERATOR_TOOLSET=       # [win]
+    - CMAKE_GENERATOR=               # [win]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clangdev
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+    - make                           # [not win]
+    - libcxx                         # [linux]
```

Comment:
This should be automatically installed by `compiler('cxx')`.

Reply:
`clang` is not our default compiler toolchain for linux. This is why the compiler directive is not installing libcxx for you.

Is there a reason that this project cannot compile with `gcc` on linux? Your addition of clangdev is adding extra dependencies that are not being used by the build or at runtime.

Reply:
I did not choose to use `clang`, it seems to be selected by default